pages/use-cases/change-data-capture.vue

```diff
@@ -46,11 +46,12 @@
             return {
                 mainContent: [
                     {
-                        title: 'CDC Implementation for Seamless Data Synchronization',
+                        title: 'Real Time Data Replication',
                         img: '/landing/usecases/cdc/feature-1.svg',
                         imgAlt: 'Diagram illustrating Kestra\'s Change Data Capture process, with PostgreSQL for databases, Apache Kafka for streaming data, and Apache Spark for analytics processing',
-                        content: 'Utilizing Kestra\'s Change Data Capture capabilities, financial institutions can detect and capture data source changes promptly.<br /><br />' +
-                            '<strong>For example:</strong> the institution uses PostgreSQL for transactional databases, Apache Kafka for streaming data, and Apache Spark for analytics processing. Kestra\'s CDC capabilities help detect changes in the PostgreSQL database and publish the changes to Kafka topics. Apache Spark then consumes the Kafka messages, processes the data, and updates the data warehouse and analytics platforms.'
+                        content: '<strong>Streamlined Data Detection and Distribution</strong>: Kestra actively listens for changes in your chosen databases, such as PostgreSQL, detecting new entries and updates as they occur. Once identified, these changes are packaged and distributed to messaging systems like Apache Kafka, guaranteeing that downstream processes, including analytics processing performed by tools like Apache Spark, have access to the latest data without delay.<br /><br />' +
```

Comment:
```suggestion
                        content: '<strong>Streamlined Data Detection and Distribution</strong>: Kestra actively listens for changes in your chosen databases, such as PostgreSQL, detecting new entries and updates as they occur. Once identified, these changes are packaged and distributed to messaging systems like Apache Kafka, guaranteeing that downstream processes have access to the latest data without delay.<br /><br />' +
```

Spark is a Hadoop Tech :) not sure if this is that important to mention

Reply:
Choose Spark because it's commonly used on big companies on legacy but sure if we want to be more "modern" I agree